stl/inc/format

```diff
@@ -1187,6 +1187,24 @@ public:
         }
     }
 
+    _NODISCARD size_t _Calculate_reserve_size() const noexcept {
+        using _CharT        = typename _Context::char_type;
+        size_t _Result      = 0;
+        const auto _Visitor = [&_Result]<class _ArgTy>(const _ArgTy _Arg) noexcept {
+            if constexpr (is_same_v<_ArgTy, basic_string_view<_CharT>>) {
+                _Result += _Arg.size();
+            } else if constexpr (is_same_v<_ArgTy, const _CharT*>) {
+                _Result += char_traits<_CharT>::length(_Arg);
+            } else {
+                _Result += 8;

```

Comment:
What's the rationale for this value of 8?

Reply:
As described in this comment https://github.com/microsoft/STL/issues/1802#issuecomment-813537549 it's a value that a similar C# feature uses and I think it might be reasonable.

Reply:
Thanks - I know the function is called `_Estimate_required_capacity` now, but I feel that these magic numbers deserve comments. With that I'm happy with the rationale.